dockerfile/torch1.14-cuda11.8.dockerfile

```diff
@@ -36,7 +36,8 @@ RUN apt-get update && \
 
 ARG NUM_MAKE_JOBS=
 ENV PATH="${PATH}" \
-    LD_LIBRARY_PATH="/usr/local/lib:${LD_LIBRARY_PATH}"
+    LD_LIBRARY_PATH="/usr/local/lib:${LD_LIBRARY_PATH}" \
+    PYTHONOPTIMIZE=1
```

Comment:
Where is this environment variable used?

Reply:
it's for python